examples/plot_simulate_alpha.py

```diff
@@ -54,18 +53,46 @@
 dpl = simulate_dipole(net, postproc=False)
 
 ###############################################################################
-# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
-# power spectral density (PSD). Note that the SciPy-function
-# `~scipy.signal.spectrogram` is used to create the plot. The
-# ``dpl[trial_idx].scale()`` call relates to the amount of cortical tissue
-# necessary to observe the electric current dipole outside the head with M/EEG.
+# Prior to plotting, we can choose to smooth the dipole waveform (note that the
+# :meth:`~hnn_core.dipole.smooth`-method operates in-place, *i.e.*, it alters
+# the data inside the ``Dipole`` object). Smoothing approximates the effect of
+# signal summation from a larger number and greater volume of neurons than are
+# included in our biophysical model; extra-cranially measured fields are always
```

Comment:
but we are not talking about extracranial fields here. We are talked about source-localized dipoles ...

Reply:
...which are localised based on the extracranial fields we measure...

I do see your point though. I'll remove the bit after the semicolon and leave it implicit here.